crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -93,6 +98,38 @@
         java.security.Security.setProperty("networkaddress.cache.ttl", "30");
     }
 
+    /**
+     * Thread-safe, shared static instance for all requests. Registry is also static, so does not
+     * make much difference to inject. If you wanted to inject, you could do so rather easily using
+     * CDI injection.
+     */
+    private static final RequestMetrics metrics =
+            new DropwizardRequestMetrics(MetricRegistryFactory.getMetricRegistry());
+
+    private static final HealthCheckRegistry healthCheckRegistry =
+            CrudCheckRegistry.getHealthCheckRegistry();
```

Comment:
Since this is static, I think you should be able to just use the `CrudCheckConfiguration` here. If I understand correctly it should still only create one instance per classloader and it removes the need for the extra class.